cfme/tests/v2v/test_v2v_migrations_single_vcenter.py

```diff
@@ -178,153 +155,109 @@ def _is_migration_started():
 
     @request.addfinalizer
     def _cleanup():
         infrastructure_mapping_collection.delete(mapping)
 
-    migration_plan_collection = appliance.collections.v2v_plans
+    migration_plan_collection = appliance.collections.v2v_migration_plans
     migration_plan = migration_plan_collection.create(
         name="long_name_{}".format(fauxfactory.gen_alphanumeric()),
         description="desc_long_name{}".format(fauxfactory.gen_alphanumeric()),
         infra_map=mapping.name,
         vm_list=[vm_obj],
-        start_migration=True,
+        target_provider=provider
     )
+    assert migration_plan.plan_started
+    assert migration_plan.in_progress
+    assert migration_plan.completed
+    assert migration_plan.successful
 
-    # explicit wait for spinner of in-progress status card
-    view = appliance.browser.create_view(
-        navigator.get_class(migration_plan_collection, "All").VIEW.pick()
-    )
-    wait_for(
-        func=view.progress_card.is_plan_started,
-        func_args=[migration_plan.name],
-        message="migration plan is starting, be patient please",
-        delay=5,
-        num_sec=150,
-        handle_exception=True,
-        fail_cond=False
-    )
-
-    # wait until plan is in progress
-    wait_for(
-        func=view.plan_in_progress,
-        func_args=[migration_plan.name],
-        message="migration plan is in progress, be patient please",
-        delay=5,
-        num_sec=1800,
-    )
-    view.switch_to("Completed Plans")
-    view.wait_displayed()
-    migration_plan_collection.find_completed_plan(migration_plan)
-    logger.info("For plan {plan_name}, migration status : {count}, total time elapsed: {clock}"
-                .format(plan_name=migration_plan.name,
-                count=view.migration_plans_completed_list.get_vm_count_in_plan(migration_plan.name),
-                clock=view.migration_plans_completed_list.get_clock(migration_plan.name)))
-
-    # validate MAC address matches between source and target VMs
-    assert view.migration_plans_completed_list.is_plan_succeeded(migration_plan.name)
-    migrated_vm = get_migrated_vm_obj(vm_obj, v2v_providers.rhv_provider)
+    migrated_vm = get_migrated_vm(vm_obj, provider)
     assert vm_obj.mac_address == migrated_vm.mac_address
 
 
-@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+@pytest.mark.parametrize('mapping_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
  indirect=True)
-def test_migration_with_edited_mapping(request, appliance, v2v_providers, edited_form_data,
-                                       form_data_vm_obj_single_datastore,
-                                       host_creds, conversion_tags, soft_assert):
+def test_migration_with_edited_mapping(request, appliance, source_provider, provider,
+                                       mapping_data_vm_obj_single_datastore):
     """
         Test migration with edited infrastructure mapping.
         Steps:
```

Comment:
@sshveta can you add this _steps_ into _teststeps_ or remove it from here. (I am doubtful if polarion docstrings takes this _steps_, I think it will ignore it by considering it as description of test title)